content/help/bugs/index.md

```diff
@@ -16,6 +16,9 @@ it. If you don't find your problem in the database, please open a [new
 issue](https://github.com/istio/istio/issues/new/choose) and let us know
 what's going on.
 
+If you think it is a security vulnerability, report it following the procedure
+in [security vulnerabilities reporting](https://istio.io/about/security-vulnerabilities/).
```

Comment:
detailed in our [security vulnerabilities reporting section](https://istio.io/about/security-vulnerabilities/).

Reply:
Done.